src/components/results/utils.js

```diff
@@ -75,7 +75,7 @@ const recursiveStringify = ( data, max = MAX_LENGTH ) => {
 	}
 
 	return {
-		length: data.toString().length,
+		length: data ? data.toString().length : 0,
```

Comment:
This is something new I had to add as it was failing.
typeof null is 'object' so it was falling back here, and data.toString() has resulted in a `null pointer exception`.

Reply:
looks like I accidentally introduced that in #91 where I didn't realize that `_.toString` checks for nullity.

I'm going to merge this PR and get it in my latest revision and hopefully we can move this forward and you can finally have the view you want in there!